_projects/syrinx.md

```diff
@@ -7,4 +7,4 @@ contributors:
     - "Raffaele Viglianti, London"
 archive: true
 ---
-_Syrinx_ (_La Flûte de Pan_) by Claude Debussy (1862 - 1918) is a short piece for flute solo originally composed as a theatrical interlude called La Flûte de Pan. Comparing the first edition (published posthumously in 1927), a manuscript discovered in 1991, a 1996 edition, and a 1922 _Music and Letters_ article by the first performer of _La Flûte de Pan_, [this report](http://legacy.cch.kcl.ac.uk/syrinx/about.html) investigates the potential of digital representation and presentation of comparative editions of music using MEI.
\ No newline at end of file
+_Syrinx_ (_La Flûte de Pan_) by Claude Debussy (1862 - 1918) is a short piece for flute solo originally composed as a theatrical interlude called La Flûte de Pan. Comparing the first edition (published posthumously in 1927), a manuscript discovered in 1991, a 1996 edition, and a 1922 _Music and Letters_ article by the first performer of _La Flûte de Pan_, [this report](https://core.ac.uk/download/pdf/147825165.pdf) investigates the potential of digital representation and presentation of comparative editions of music using MEI.
```

Comment:
```suggestion
_Syrinx_ (_La Flûte de Pan_) by Claude Debussy (1862 - 1918) is a short piece for flute solo originally composed as a theatrical interlude called La Flûte de Pan. Comparing the first edition (published posthumously in 1927), a manuscript discovered in 1991, a 1996 edition, and a 1922 _Music and Letters_ article by the first performer of _La Flûte de Pan_, [this report](https://core.ac.uk/download/pdf/147825165.pdf#page=41) investigates the potential of digital representation and presentation of comparative editions of music using MEI.
```

Reply:
However, I am not sure if that's the exact target that was addressed by the broken link: http://legacy.cch.kcl.ac.uk/syrinx/about.html

Maybe @raffazizzi can confirm?

Reply:
The original address was [archived](https://web.archive.org/web/20100712140310/http://legacy.cch.kcl.ac.uk/syrinx/about.html), but the _Digital Edition Demo_ doesn't work anymore. The original report can be found here: http://dh2010.cch.kcl.ac.uk/academic-programme/abstracts/papers/html/ab-819.html 
There's also a PDF version available.